src/udev/udevd.c

```diff
@@ -1448,10 +1448,11 @@ static int on_sigchld(sd_event_source *s, const struct signalfd_siginfo *si, voi
                         device_tag_index(worker->event->dev, NULL, false);
 
                         if (manager->monitor) {
-                                /* forward kernel event without amending it */
+                                /* forward kernel event without amending anything */
```

Comment:
Hmm, "amending anything" is a bit strange. We generally use "amend" as "change-in-place". "amend it" is clear — the message is not modified, more than "amending anything". Maybe this should be "Forward kernel event unchanged" ?